standalone/pherry/src/headers_cache.rs

```diff
@@ -179,7 +179,7 @@ async fn grab_headers(
 
     let header_hash = crate::get_header_hash(api, Some(start_at - 1)).await?;
     let mut last_set = api.current_set_id(Some(header_hash)).await?;
-    let mut skip_justitication = justification_interval;
+    let mut skip_justitication = 0_u32;
```

Comment:
why hardcode?

Reply:
We just let it start with 0.

The logic is:
We grab headers in a loop and decrease the `skip_justification` by 1 in each round. When it is decreased to `0`, the justification in the header would be kept, removed otherwise.

So, if we start with a `0`, it would keep the first justification. 
```
before:   --------------*-------------*---------------*-......
now:       -*------------*-------------*---------------*-......
            ^
      justification
```